### ISSUE TITLE
Segfault on hackrf() call if not connected to device

### ISSUE THREAD

**Issue by bergie** [mappum/node-hackrf#1]
If one tries to use this module without a connected HackRF device, it crashes.

```
1]    52545 segmentation fault  ./node_modules/.bin/noflo-nodejs
```

**Comment by max-mapper** [comment]
we'd have to use some libhackrf API to find out the USB state... as it uses libusb directly and we don't bind to it

**Comment by mafintosh** [comment]
hackrf has the information already - we just need to check how many devices are connected

**Comment by mafintosh** [comment]
fixed in 2.2.0
